src/Client.php

```diff
@@ -143,9 +143,8 @@ public function __construct(?string $uri = null, array $uriOptions = [], array $
      * Return internal properties for debugging purposes.
      *
      * @see https://php.net/manual/en/language.oop5.magic.php#language.oop5.magic.debuginfo
-     * @return array
```

Comment:
Technically, yes. However, since this method is only used when calling `var_dump` and we don't want users to calling it manually, I don't see the benefit of properly defining the array shape for static analysis.